doc/classes/AudioStreamGeneratorPlayback.xml

```diff
@@ -14,18 +17,21 @@
 			<argument index="0" name="amount" type="int">
 			</argument>
 			<description>
+				Returns [code]true[/code] if a buffer of the size [code]amount[/code] can be pushed to the audio sample data buffer without overflowing it, [code]false[/code] otherwise.
 			</description>
 		</method>
 		<method name="clear_buffer">
 			<return type="void">
 			</return>
 			<description>
+				Clears the audio sample data buffer.
 			</description>
 		</method>
 		<method name="get_frames_available" qualifiers="const">
 			<return type="int">
 			</return>
 			<description>
+				Returns the number of audio data frames left to play. If this returned number reaches [code]0[/code], the audio will stop playing until frames are added again. Therefore, make sure your script can always generate and push new audio frames fast enough to avoid audio cracking.
 			</description>
 		</method>
 		<method name="get_skips" qualifiers="const">
```

Comment:
Does anyone know what this method is used for? I'd like to get 100% completion :slightly_smiling_face: